iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/audit/AbstractAuditLogger.java

```diff
@@ -47,4 +48,16 @@ public abstract class AbstractAuditLogger {
   public boolean noNeedInsertAuditLog(IAuditEntity auditLogFields) {
     return true;
   }
+
+  public void recordAuditLog(final IAuditEntity auditEntity, final Supplier<String> auditObject) {
```

Comment:
I don't think it is a good idea that roughly moving the `recordAuditLog` function from AuthCheckImpl to AbstractAuditLogger, because the current implementation is only for `OBJECT_AUTHENTICATION` audit. If u indeed require this refactor, I know current change is for pipe audit, at least rename this interface to `recordObjectAuthenticationAuditLog`.